### ISSUE TITLE
 xxx is not a valid Python package name

### ISSUE THREAD

**Issue by long2ice** [leinardi/mypy-pycharm#95]
After upgrade, there is a error occur

<img width="792" alt="image" src="https://user-images.githubusercontent.com/13377178/156095465-9e3aac03-5942-4baf-b150-e9536b6f0643.png">

**Comment by intgr** [comment]
This error message is verbatim from mypy. Previous plugin versions simply ignored these errors, but they are reported since 0.14.0.

See https://github.com/python/mypy/issues/8400 https://github.com/python/mypy/issues/10515, looks like you need to change `__init__.py` files in your project. Maybe `--namespace-packages` flag also affects this behavior, I am not sure.

**Comment by long2ice** [comment]
OK, thanks for you reply

**Comment by intgr** [comment]
Did this solve the problem you had? It's possible that the real issue you are having is #95. I haven't yet looked into it in depth though.

**Comment by Andrioden** [comment]
Having the same problem, no idea what `csi-015` is. The most annoying part is that i get these error popups.

Cant find any files/paths with csi in it either: `ls -a -R | grep csi`

Running `mypy .` and `dmypy run .` works just fine.

My suggestion is that you @intgr either print more output in the error as I have no idea what is failing, or hide the warnings.

```

07.04.2022
00:51	Mypy Plugin (Mypy exited abnormally): csi-015 is not a valid Python package name

00:51	Unexpected Exception Caught
			The scan failed due to an exception: Mypy failed with code 2
			com.leinardi.pycharm.mypy.exception.MypyToolException: Mypy failed with code 2
			at com.leinardi.pycharm.mypy.mpapi.MypyRunner.runMypy(MypyRunner.java:324)
			at com.leinardi.pycharm.mypy.mpapi.MypyRunner.scan(MypyRunner.java:260)
			at com.leinardi.pycharm.mypy.checker.ScanFiles.scan(ScanFiles.java:109)
			at com.leinardi.pycharm.mypy.checker.ScanFiles.checkFiles(ScanFiles.java:100)
			at com.leinardi.pycharm.mypy.checker.ScanFiles.call(ScanFiles.java:74)
			at com.leinardi.pycharm.mypy.MypyAnnotator.doAnnotate(MypyAnnotator.java:131)
			at com.leinardi.pycharm.mypy.MypyAnnotator.doAnnotate(MypyAnnotator.java:62)
			at com.intellij.codeInsight.daemon.impl.ExternalToolPass.doAnnotate(ExternalToolPass.java:218)
			at com.intellij.codeInsight.daemon.impl.ExternalToolPass.doAnnotate(ExternalToolPass.java:212)
			... (show balloon)

00:51	Mypy Plugin (Mypy exited abnormally): csi-017 is not a valid Python package name

00:51	Unexpected Exception Caught
			The scan failed due to an exception: Mypy failed with code 2
			com.leinardi.pycharm.mypy.exception.MypyToolException: Mypy failed with code 2
			at com.leinardi.pycharm.mypy.mpapi.MypyRunner.runMypy(MypyRunner.java:324)
			at com.leinardi.pycharm.mypy.mpapi.MypyRunner.scan(MypyRunner.java:260)
			at com.leinardi.pycharm.mypy.checker.ScanFiles.scan(ScanFiles.java:109)
			at com.leinardi.pycharm.mypy.checker.ScanFiles.checkFiles(ScanFiles.java:100)
			at com.leinardi.pycharm.mypy.checker.ScanFiles.call(ScanFiles.java:74)
			at com.leinardi.pycharm.mypy.MypyAnnotator.doAnnotate(MypyAnnotator.java:131)
			at com.leinardi.pycharm.mypy.MypyAnnotator.doAnnotate(MypyAnnotator.java:62)
			at com.intellij.codeInsight.daemon.impl.ExternalToolPass.doAnnotate(ExternalToolPass.java:218)
			at com.intellij.codeInsight.daemon.impl.ExternalToolPass.doAnnotate(ExternalToolPass.java:212)
			... (show balloon)

```

![image](https://user-images.githubusercontent.com/393376/162086717-56aa1cd7-eb0b-4f3c-8f6f-553a86ad953d.png)

**Comment by zhukovgreen** [comment]
In my case the problem was solved, when I `rm -frv .mypy_cache`
